hw/system/occamy/test/occamy.dts

```diff
@@ -42,12 +42,6 @@
     #size-cells = <2>;
     compatible = "simple-bus";
     ranges;
-    debug@0 {
```

Comment:
The debug module description seems to have disappeared entirely from the device tree; can you double-check this?

Reply:
oh good catch! I will double check

Reply:
Yes, the `debug` entry is deprecated: https://github.com/openhwgroup/cva6/pull/806

Reply:
In that case, we should also remove it from the generator code.

Also, looking at the UART fix in that PR, this also directly concerns us, right @niwis?

Reply:
Yes, indeed. I would cherry-pick the PR once it is merged.

Reply:
I fixed it to the previous state (with debug module). The other CVA6 and linux booting related changes (remove debug and fix the UART) should be done in another PR for keeping it easier trackable.